pennylane/workflow/qnode.py

```diff
@@ -628,6 +630,67 @@ def add_transform(self, transform_container: TransformContainer):
         """
         self._transform_program.push_back(transform_container=transform_container)
 
+    def update(self, **kwargs) -> "QNode":
+        """Return a new QNode instance with updated constructor arguments.

```

Comment:
```suggestion
        """Returns a new, identical QNode instance but with updated settings (e.g., a different `diff_method`). Any settings not specified will retain their original value.
```

"Identical" might not be the best word here, but I'm trying to make sure that the user knows that "everything is the same except for what you specify to change" 🤔